src/patterns/BidSnapshot/BidSnapshot.tsx

```diff
@@ -0,0 +1,147 @@
+import { ComponentProps, forwardRef } from 'react';
+import classnames from 'classnames';
+
+import { getCommonProps } from '../../utils';
+import { DetailList } from '../DetailList/index';
+import { Detail } from '../../components/Detail/index';
+import { AuctionStatus, SupportedLanguages } from '../../types/commonTypes';
+import { Countdown } from '../../components/Countdown/index';
+import { CountdownVariants } from '../../components/Countdown/types';
+
+export interface BidSnapshotProps extends ComponentProps<'div'> {
+  /**
+   * Active bid of current user. -  '$1,000'
+   */
+  activeBid?: string;
+  /**
+   * State of the object
+   */
+  auctionStatus?: AuctionStatus;
```

Comment:
I'm wondering if this particular component should be in `remix` because it could reuse the actual API `AuctionStateEnum`.